### DISCUSSION TITLE
PBIX Xpress9 Decompression to PBIT / Metadata information

### DISCUSSION THREAD

**Discussion by justmike1** [TabularEditor/TabularEditor3#1355] [🙋♀️TE3 Q&A] [answered]
Hey guys, 

I am found myself in this github repository after trying to find a way of how Microsoft handles the `pbix` compression to try to extract information out of it from the `DataModel` file inside.

Although I found nothing, I came across this tool which supports importing a `pbix` file, and when importing, it is also showcasing the tables inside and such, so although this tool does the solution, because it is open source I would like if possible to share the piece of code which successfully imports a `pbix` file to the tool, or it has some other workaround which isn't from the `DataModel` file...

**Comment by otykier** [comment]
We do not support pbix files as that file format is proprietary. But you could try to talk to [this guy](https://x.com/igocrite/status/1774722314195669054?s=46&t=HULt15koKFdmZMW0nbi_vQ).

**Reply by justmike1** [reply]
so how does the import button works for PBIX? or it doesn't?

**Reply by justmike1** [reply, marked as answer]
I found it using pbi-tools

**Reply by justmike1** [reply]
https://pbi.tools/cli
